src/components/messages/warning-message/README.md

```diff
@@ -0,0 +1,23 @@
+# WarningMessage
+
+#### Description
+
+Represents an warning message.
+
+## Usage
+
+```js
+import WarningMessage from '@commercetools-frontend/ui-kit/messages/error-message';
+
+<WarningMessage>This is a duplicate of {`${duplicate}`}</WarningMessage>;
+```
+
+#### Properties
+
+| Props      | Type   | Required | Values | Default | Description                               |
+| ---------- | ------ | :------: | ------ | ------- | ----------------------------------------- |
+| `children` | `node` |    ✅    | -      | -       | Warning message, either as string or node |
```

Comment:
Sidenote: The bug in the library used by `markdown-loader` where empty cells were not possible was now fixed. We can use empty cells again. I didn't try it yet, but I read it in the release notes. So we don't need to fill empty cells with `-` anymore.